thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/EmailResource.java

```diff
@@ -181,6 +187,81 @@ public Response updateAlertConfigHolidayCron(@NotNull @PathParam("id") Long id,
     return getAlertToSubscriberMapping().get(alertFunctionId);
   }
 
+  @POST
+  @Path("{id}/subscribe")
+  @ApiOperation("Endpoint for subscribing to an alert/alerts")
+  public Response subscribeAlert(
+      @PathParam("id") Long id,
+      @QueryParam("functionId") String functionIds,
+      @QueryParam("functionName") String functionName,
+      @QueryParam("metric") String metric,
+      @QueryParam("dataset") String dataset,
+      @DefaultValue("") @QueryParam("functionPrefix") String functionPrefix) throws Exception {
+    if (org.apache.commons.lang3.StringUtils.isBlank(functionIds) && org.apache.commons.lang3.StringUtils.isBlank(functionName)
```

Comment:
import the base namespace. Both lang and lang3 should work here anyways

Reply:
Aah! that must have happened while copy-pasting it to a different resource file. Will fix it.